model/system/metrics.yaml

```diff
@@ -1,4 +1,17 @@
 groups:
+  # system.* metrics
+  - id: metric.system.uptime
+    type: metric
+    metric_name: system.uptime
+    stability: experimental
+    brief: "The time the system has been running"
```

Comment:
Based on the response to my earlier question to semconv maintainers, I think this brief should be adjusted that the time is a floating point number of seconds.

Reply:
updated